backend/services/team_service.py

```diff
@@ -302,17 +314,9 @@ def get_team_as_dto(team_id: int, user_id: int, abbreviated: bool) -> TeamDTO:
         if abbreviated:
             return team_dto
 
-        team_members = TeamService._get_team_members(team_id)
-        for member in team_members:
-            user = UserService.get_user_by_id(member.user_id)
-            member_dto = TeamMembersDTO()
-            member_dto.username = user.username
-            member_dto.pictureUrl = user.picture_url
-            member_dto.function = TeamMemberFunctions(member.function).name
-            member_dto.picture_url = user.picture_url
-            member_dto.active = member.active
-
-            team_dto.members.append(member_dto)
+        team_dto.members = [
+            TeamService.get_team_member_as_dto(member) for member in team.members
+        ]
 
         team_projects = TeamService.get_projects_by_team_id(team.id)
         for team_project in team_projects:
```

Comment:
Similar to `TeamService.get_team_member_as_dto(member)` can we move the `TeamProjectDTO` to a modular function? It will be useful so that we could reuse elsewhere if needed